src/MultipartStreamBuilder.php

```diff
@@ -149,13 +149,13 @@ public function build()
             }
             if ($contentStream->isReadable()) {
                 while (!$contentStream->eof()) {
-                    // read 1MB chunk into buffer until reached EOF.
+                    // Read 1MB chunk into buffer until reached EOF.
                     fwrite($buffer, $contentStream->read(1048576));
                 }
             } else {
-                // Try to getContents for non-readable stream.
-                // Less controllable chunk size (thus memory usage).
-                fwrite($buffer, $contentStream->getContents());
+                // If not isReadable, read from __toString().
+                // Warning: This could attempt to load a large amount of data into memory.
```

Comment:
Don't see the point in this warning? Who is the comment intended for? End users will not read this.

Reply:
I, as end user, would read code from time to time. But if it makes you happy, I can remove the warning line.

Reply:
How else, if needed, do you want me to change it?

Reply:
Yes, lets remove both these comments. 

```
// If not isReadable, read from __toString().
// Warning: This could attempt to load a large amount of data into memory.
```